release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -84,107 +46,37 @@ This document lists known issues for **.NET Core 3.0 Preview releases** which ma
-  __webpack_require__
-  ClientApp/webpack/bootstrap 43633f0e2b726d97cc14:678
-  fn
-  ClientApp/webpack/bootstrap 43633f0e2b726d97cc14:88
-  0
-  https://localhost:44315/static/js/bundle.js:54860:18
-  __webpack_require__
-  ClientApp/webpack/bootstrap 43633f0e2b726d97cc14:678
-  (anonymous function)
-  ClientApp/webpack/bootstrap 43633f0e2b726d97cc14:724
-  (anonymous function)
-  https://localhost:44315/static/js/bundle.js:728:10
-  ```
-
-  * Workaround:
-    * Remove the file /ClientApp/src/components/api-authorizationApi/AuthorizationRoutes.js
-    * Replace the contents of App.js with these
-      ```javascript
-      import React, { Component } from 'react';
-      import { Route } from 'react-router';
-      import { Layout } from './components/Layout';
-      import { Home } from './components/Home';
-      import { FetchData } from './components/FetchData';
-      import { Counter } from './components/Counter';
-      import { Login } from './components/api-authorization/Login'
-      import { Logout } from './components/api-authorization/Logout'
-      import AuthorizeRoute from './components/api-authorization/AuthorizeRoute';
-      import { ApplicationPaths, LoginActions, LogoutActions } from './components/api-authorization/ApiAuthorizationConstants';
-
-      export default class App extends Component {
-        static displayName = App.name;
-
-        render () {
-          return (
-            <Layout>
-              <Route exact path='/' component={Home} />
-              <Route path='/counter' component={Counter} />
-              <AuthorizeRoute path='/fetch-data' component={FetchData} />
-              <Route path={ApplicationPaths.Login} render={() => loginAction(LoginActions.Login)} />
-              <Route path={ApplicationPaths.LoginFailed} render={() => loginAction(LoginActions.LoginFailed)} />
-              <Route path={ApplicationPaths.LoginCallback} render={() => loginAction(LoginActions.LoginCallback)} />
-              <Route path={ApplicationPaths.Profile} render={() => loginAction(LoginActions.Profile)} />
-              <Route path={ApplicationPaths.Register} render={() => loginAction(LoginActions.Register)} />
-              <Route path={ApplicationPaths.LogOut} render={() => logoutAction(LogoutActions.Logout)} />
-              <Route path={ApplicationPaths.LogOutCallback} render={() => logoutAction(LogoutActions.LogoutCallback)} />
-              <Route path={ApplicationPaths.LoggedOut} render={() => logoutAction(LogoutActions.LoggedOut)} />
-            </Layout>
-          );
-        }
-      }
+- **Server-side Blazor app fails to run on non-Windows platforms**
+
+    Running a server-side Blazor app on a non-Windows platform fails with the following error:
+
+    ```
+    System.InvalidOperationException: Could not load the embedded file manifest 'Microsoft.Extensions.FileProviders.Embedded.Manifest.xml' for assembly 'Microsoft.AspNetCore.Components.Server'.
```

Comment:
Would you like me to investigate possible workarounds for this?

Reply:
That would be lovely